File: src/primus/pawel/Exercise_184.java
package primus.pawel;

import java.util.regex.Matcher;
import java.util.regex.Pattern;

// Count the Characters
// 7kyu
// https://www.codewars.com/kata/577ad961ae2807182f000c29/train/java
// 2021-07-19 T:22:38:07
class Exercise_184 {
    public static int charCount(String str, char c) {
        Pattern p = Pattern.compile(String.valueOf(c).toLowerCase());
        Matcher m = p.matcher(str.toLowerCase());

        return (int) m.results().count();
    }

}

File: src/JS/Exercise_184.js
// Check for prime numbers
// 7kyu
// https://www.codewars.com/kata/53daa9e5af55c184db00025f/train/javascript
// 2022-07-23 T:11:38:42

function isPrime(n) {
    if (n < 2) {
        return false;
    }
    if (n === 2) {
        return true;
    }
    if (n % 2 === 0) {
        return false
    }

    for (let i = 3; i < n; i = i + 2) {
        if (n % i === 0) {
            return false;
        }
    }
    return true;
}